README.md

```diff
@@ -0,0 +1,80 @@
+# ReactJS Boilerplate
+
+## Roadmap
+
+Our roadmap for the ReactJS Boilerplate is where you can learn about what features we're working on.
+
+### Development environment
+
+- Build tool
+  - [x] Vite
+    - Optimisation of Minify code in production mode
+- Coding Standards and Guidelines (Statically analyzes our codes)
+  - [ ] ESLint with our rules
+  - [ ] Prettier
+- Testing framework
+  - [x] Jest
+  - [ ] React Testing Library
+  - [ ] React Hooks Testing Library
+  - [ ] Storybook
+    - [x] base
+    - [ ] msw
+    - [ ] state management
+    - [ ] router
+    - [ ] css
+    - [ ] i18n
+- Mock API server
+  - [x] msw
+
+### Application design
+
+- State management
+  - Client state
+    - [ ] useContext
+      - Need to have logic to prevent unnecessary rerender
+    - [ ] Use state management libraries
+      - [x] zustand
+      - Continued consideration of other proposals
+  - Server state
+    - [x] React Query
+  - Typescript rules
+    - [ ] [Interface vs Types](https://github.com/monstar-lab-oss/reactjs-boilerplate/discussions/20)
+- Routing for SPA
+  - [x] [wouter](https://github.com/molefrog/wouter)
+- Validate forms in React
+  - [x] [React Hook Form](https://react-hook-form.com)
```

Comment:
Ignore this comment if it has been discussed already...

How about Formik and Yup? It is common using this combination for forms

Reply:
@yukinoda 
Sorry for the late reply. I adopted react-hook-form for its ability to [prevent unnecessary re render](https://codesandbox.io/s/react-hook-form-js-wbybv?from-embed), does Formik have the same support? and I think react-hook-form can also do Yup and combination.

```js
const validationSchema = yup.object().shape({
  email: yup.string().required('error message').email('error message'),
  password: yup.string().required('error message').min(6, 'error message')
})

const { register, setValue, handleSubmit, errors } = useForm({ validationSchema })
```